src/locale/pt-br.js

```diff
@@ -3,6 +3,7 @@ import dayjs from 'dayjs'
 const locale = {
   name: 'pt-br',
   weekdays: 'Domingo_Segunda-feira_Terça-feira_Quarta-feira_Quinta-feira_Sexta-feira_Sábado'.split('_'),
+  weekdaysShort: 'Dom._Seg._Ter._Qua._Qui._Sex._Sáb.'.split('_'),
```

Comment:
thanks, how about removing all the ending dot to keep the same with 
https://github.com/moment/moment/blob/develop/locale/pt-br.js

Reply:
Normally we use "." at the end of this weekdays.
But no problem, I'll fix it.